rclcpp/include/rclcpp/logger.hpp

```diff
@@ -161,6 +162,10 @@ class Logger
   RCLCPP_PUBLIC
   void
   set_level(Level level);
+
+  RCLCPP_PUBLIC
+  rcpputils::fs::path
+  get_logging_directory() const;
```

Comment:
It's probably worth adding some API docs similar to [rcl_get_logging_directory](https://github.com/ros2/rcl_logging/blob/a1edc81af3a7ac87fc63ce36bd890e7000dba7b7/rcl_logging_interface/include/rcl_logging_interface/rcl_logging_interface.h#L95-L103)

Reply:
Added in https://github.com/ros2/rclcpp/pull/1509/commits/1b6c60f52776d92f7966bd4210138cea76067333.